SPEED_VS_SOVEREIGNTY.md

```
@@ -15,18 +17,18 @@ However, during this hackathon, we face a hard constraint: **time**.
 | Embeddings | SentenceTransformer | Local CPU | ~2-13 sec/chunk |
 | LLM Extraction | Local LLM / Maple | Self-hosted | ~10-30 sec/chunk |
 | Vector Store | Qdrant | Local container | Fast |
-| Graph Store | Neo4j | Local container | Fast |
+| Graph Store | Neo4j | Local container | Fast *(planned)* |
 
 **Total for 150-chunk document: 30-60+ minutes**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)